dts/xtensa/espressif/esp32.dtsi

```diff
@@ -81,6 +86,12 @@
 			};
 		};
 
+		ipm0: ipm@3ffe5a30 {
+			compatible = "espressif,esp32-ipm";
+			reg = <0x3FFE5A30 0x8>;
+			ipm_channels = <4>;
```

Comment:
If the driver requires channels I strongly suggest to move this driver to the MBOX API instead.

Reply:
No, it has a single channel only, the initial design featured multiple channels and this property was a result.

Should I update this file?

Reply:
Yes. Remove it if you do not use the property.